OpenSim/Examples/ControllerExample/tugOfWar_model_ThelenOnly.osim

```diff
@@ -281,10 +281,6 @@
 				<Thelen2003Muscle name="muscle1">
 					<!--Flag indicating whether the force is disabled or not. Disabled means that the force is not active in subsequent dynamics realizations.-->
 					<isDisabled>false</isDisabled>
-					<!--Minimum allowed value for control signal. Used primarily when solving for control values.-->
```

Comment:
Do any tests cover the cases where (i) `0 < min_control < 0.01` or (ii) `0.01 < min_control < 1` to ensure the user-supplied `min_control` isn't just being ignored/overwritten?